src/featbin/append-ivector-to-feats.cc

```diff
@@ -0,0 +1,231 @@
+// featbin/append-ivector-to-feats.cc
```

Comment:
I think this file is probably not necessary; the nnet2 scripts already do this task based on existing binaries.